app/models/sections/items/ItemWineGrowingZone.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.sections.items
+
+import models.{Enumerable, WithName}
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.Aliases.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.RadioItem
+
+sealed trait ItemWineGrowingZone
+
+object ItemWineGrowingZone extends Enumerable.Implicits {
+
+  case object A      extends ItemWineGrowingZone
+  case object B      extends ItemWineGrowingZone
+  case object CI     extends ItemWineGrowingZone
+  case object CII    extends ItemWineGrowingZone
+  case object CIII_A extends ItemWineGrowingZone
+  case object CIII_B extends ItemWineGrowingZone
```

Comment:
I assume these don't need `WithName("..")`?

Reply:
No, as in this case the objects will go out as `A`, `B`, CI` etc... anyway. They're already keys in a way. It seemed pointless to then add `withName` and set it to `a`, `b`, `c`...